src/qt/walletframe.cpp

```diff
@@ -99,8 +99,10 @@ bool WalletFrame::removeWallet(const QString &name)
 void WalletFrame::removeAllWallets()
 {
     QMap<QString, WalletView*>::const_iterator i;
-    for (i = mapWalletViews.constBegin(); i != mapWalletViews.constEnd(); ++i)
+    for (i = mapWalletViews.constBegin(); i != mapWalletViews.constEnd(); ++i) {
         walletStack->removeWidget(i.value());
+        delete i.value();
```

Comment:
```suggestion
       i.value()->deleteLater();
```

Not sure if it matters at all when its removed from the parent, but maybe like ^ ?

Reply:
`removeAllWallets` is called from `requestShutdown` https://github.com/dashpay/dash/blob/develop/src/qt/dash.cpp#L431 which is called outside of the main event loop https://github.com/dashpay/dash/blob/develop/src/qt/dash.cpp#L818. It doesn't look like a good place to use `deleteLater` unless I'm misreading this https://doc.qt.io/qt-5/qobject.html#deleteLater.